src/interface/src/app/services/treatment-to-pdf.service.ts

```diff
@@ -0,0 +1,179 @@
+import { Injectable } from '@angular/core';
+import jsPDF from 'jspdf';
+import autoTable from 'jspdf-autotable';
+import {
+  PRESCRIPTIONS,
+  PrescriptionSequenceAction,
+  nameForTypeAndAction,
+} from '../treatments/prescriptions';
+import { Map as MapLibreMap } from 'maplibre-gl';
+import { logoImg } from '../../assets/base64/icons';
+import { TreatmentSummary, Prescription, TreatmentProjectArea } from '@types';
+import { TreatmentsState } from '../treatments/treatments.state';
+import { TreatedStandsState } from '../treatments/treatment-map/treated-stands.state';
+@Injectable({
+  providedIn: 'root',
+})
+export class TreatmentToPDFService {
+  constructor(
+    private treatmentsState: TreatmentsState,
+    private treatedStandsState: TreatedStandsState
+  ) {}
+
+  parentMap!: MapLibreMap;
```

Comment:
maybe this should be null by default? its not set on the constructor so this might be null